cbs.py

```diff
@@ -71,44 +81,37 @@ async def on_ready():
 
 @CLIENT.event
 async def on_message(message):
-    global last_cbs_mention_details
 
     # Always ignore bot messages
     if message.author.bot:
         return
 
-    guild_id = message.guild.id
     # Uncomment to temporarily disable the bot from messaging the Minnesota Rhythm Gaming Discord Server
-    # if guildId == 190994300354560010:
+    # if message.guild.id == 190994300354560010:
     #     return
 
     # Check for a match, if it matches, send an appropriate message
     if is_match(message):
         # Save basic details about the message
-        this_cbs_mention = {"message_id": message.id, "message": message.content, "author_id": message.author.id,
-            "author": message.author.display_name, "author_username": message.author.name, "date": datetime.datetime.now()}
-
-        if str(guild_id) in last_cbs_mention_details:
+        this_cbs_message = message
+        if MESSAGE_COLLECTION.count_documents({}) > 0:
             # If we've seen someone mention combo based scoring before, then get the last time, find the timespan between now
             # and the last time it was seen in that particular Discord server, and print it out to the user
-            cbs_timespan = this_cbs_mention["date"] - last_cbs_mention_details[str(guild_id)]["date"]
+            last_cbs_message = MESSAGE_COLLECTION.find().sort({"created_at": -1}).limit(1).next()
```

Comment:
forgot to filter by guild ID, but fine with proceeding as a quick fix can be made + no effect on data.